style.css

```diff
@@ -0,0 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: calc(100vh - 44px);
+  background-image: url("./images/header-illsutration-mobile@2x.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
```

Comment:
Kindly you need to review the styling for the background image because when I scroll down the text goes down but the background image is not. 
So maybe you need to remove this role `background-attachment: fixed;`, and add `background-size: cover;` to make it cover the section.